README.md

```diff
@@ -111,6 +112,17 @@ Supports standard VS Code formatting.
 
 ![Code Format](image/codeFormat.jpg)
 
+### Align Selected Variables Assignment
+
+Select variables assignment:
+
+-   Right-click and select `Align Selected Variables Assignment`.
+-   Or press `Shift+Win+F`.
+
+![Align Assignment](image/align.gif)
+
+Comments indentation are preserved during alignment. You can add new variable assignment, re-align all assignments. Your manually align comments will stay in old place.
+
```

Comment:
```suggestion
### Align Selected Assignments

Select assignments:

-   Right-click and select `Align Selected Assignments` from the context menu.
-   Or press `Shift + Win + F`.

![](image/align.gif)

Comments' indentations are preserved during alignment. You can add new variable assignment and re-align all assignments. Your manually aligned comments will stay in their original place.

```

Reply:
Can you provide an example of "You can add new variable assignment and re-align all assignments. Your manually aligned comments will stay in their original place."?

Reply:
Why you edit this `![Align Assignment](image/align.gif)` to this `![](image/align.gif)`? All images in README.md has description in brackets.

Reply:
Descriptions are for screen reader users, yes. But `align assignment` does not provide any information as a description. Since the feature is already described in text above, it doesn't need a description. In fact, removing the description makes screen readers less chatty and provides a better user experience.

Reply:
This doesn't really matter too much, if you want to ignore the change you're welcome to.